File: app/src/main/java/com/gmail/mostafa/ma/saleh/yawa/utilities/YawaApp.java
package com.gmail.mostafa.ma.saleh.yawa.utilities;

import android.app.Application;

/**
 * Created by Mostafa on 11/11/2017.
 */

public class YawaApp extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        SharedPreferencesManager.init(this);
    }
}

File: app/src/main/java/com/gmail/mostafa/ma/saleh/yawa/utilities/YawaApp.kt
package com.gmail.mostafa.ma.saleh.yawa.utilities

import android.app.Application

/**
 * Created by Mostafa on 11/11/2017.
 */

class YawaApp : Application() {
    override fun onCreate() {
        super.onCreate()
        SharedPreferencesManager.init(this)
    }
}
